qemu/tests/balloon_hotplug.py

```diff
@@ -38,7 +39,7 @@ def run_pm_test(pm_test, plug_type):
                               % (pm_test, plug_type), logging.info)
         utils_test.run_virt_sub_test(test, params, env, pm_test)
 
-    def enable_balloon_service():
+    def enable_balloon_service(ballloon_test):
```

Comment:
I guess we don't need to pass param "balloon_test" here, it could be used directly. Otherwise a doc string is needed.

Reply:
Modified, thanks